now.json

```diff
@@ -1,4 +1,7 @@
 {
   "name": "nteract-site",
-  "alias": "nteract.io"
+  "alias": "nteract.io",
+  "env": {
+    "GH_TOKEN": "@gh-token-public"
```

Comment:
I love the use of a token & zeit secret. We'll now need to be plenty mindful before merging things.